src/redux/categories/catSlice.jsx

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  status: 'under-development',
+};
+
+const catSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    setUnderDevelopment: (state) => {
+      state.status = 'under-development';
+    },
+  },
```

Comment:
- [ ] Please make sure to replace this reducer with check status reducer as requested in the project requirements. also, remove all other files in this folder except the catogies slice. The Redux directory should only be responsible for redux code.
  - You can consider the below code.

```javascript
reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
```